File: MajorityElement.cpp
#include<iostream>
using namespace std;

int main(){

    int arr[] = {1,2,2,6,4,6};
    int n = sizeof(arr)/sizeof(arr[0]);

    for(int val : arr){
        int frq=0;
        for(int el : arr){
            if(el==val){
            frq++;
            }
        }
        if(frq>n/2)
            cout<<val;
    }
}

File: PYTHON/ARRAY/majorityelement.py
def majority(arr):

    n = len(arr)
    for val in arr:
        frq = 0
        for el in arr:
            if el == val:
                frq += 1

        if frq >= n//2:
            return val

    return -1

num = [1,2,2,2,2,3,6,8,9]
print(majority(num))